app/models/team.rb

```diff
@@ -2,9 +2,10 @@
 class Team < ApplicationRecord
   include ProfilesHelper, HasSeason
 
-  delegate :sponsored?, to: :kind
+  delegate :full_time?, to: :kind
```

Comment:
I'm sorry, but I think I don't understand what this should be 🙈

Reply:
These allow us to call `full_time?` or `part_time?` on the team models. Removing them would give me a no method error.